README.md

```diff
@@ -13,9 +12,11 @@
 ![](https://user-images.githubusercontent.com/710847/80649891-dab42300-8a40-11ea-96ac-f76926f1b109.png)
 
 ## ℹ️ How to use this module
-1. clone this repo into your moducles directory with the following command: `git clone https://github.com/Elaniobro/MMM-nyc-transit.git`
-1. install all the npm modules with either `yarn install` or `npm install`
-1. update your [Magic Mirror Config](https://github.com/MichMich/MagicMirror/blob/master/config/config.js.sample), by adding the following object:
+
+1. clone this repo into your moducles directory with the following command: `git clone https://github.com/Elaniobro/MMM-nyc-transit`
+2. install all the npm modules with either `yarn install` or `npm install`
+3. update your [Magic Mirror Config](https://github.com/MichMich/MagicMirror/blob/master/config/config.js.sample), by adding the following object:
```

Comment:
Should this also read as `MagicMirror²` now?

Reply:
Maybe rephrasing the whole line would be good? Like "Add the module to your `config.js`:"

Reply:
Do you think it _could/would_ add confusion to the customer of the module, thinking that there _might_ be a `MMM-nyc-tranist` config file? If so, lets just update it to read:
`update your [MagicMirror² Config](https://github.com/MichMich/MagicMirror/blob/master/config/config.js.sample), by adding the following object:`

Reply:
Okay, I just changed it to MagicMirror² :slightly_smiling_face: